src/commands/deploy.ts

```diff
@@ -123,54 +127,59 @@ export default class Deploy extends Command {
 
     const aquaCli = await initAquaCli(this);
     const tmpDeployJSONPath = await ensureFluenceTmpDeployJsonPath();
-    const appConfig = await initAppConfig(this);
-
-    if (appConfig !== null) {
-      // Prompt user to remove previously deployed app if
-      // it was already deployed before
-      const doRemove =
-        flags[FORCE_FLAG_NAME] ||
-        (await confirm({
-          message: `Previously deployed app described in ${color.yellow(
-            replaceHomeDir(appConfig.$getPath())
-          )} needs to be removed to continue. Do you want to remove?`,
-          isInteractive,
-          flagName: FORCE_FLAG_NAME,
-        }));
-
-      if (!doRemove) {
-        this.error("You have to confirm to continue");
-      }
-
-      await removeApp({
+    let appConfig = await initAppConfig(this);
+
+    if (
+      appConfig !== null &&
+      (isInteractive
+        ? await confirm({
+            isInteractive,
+            message:
+              "Do you want to remove some of the previously deployed services?",
```

Comment:
Can we specify the exact services that are going to be removed?

Reply:
If you answer yes, then it will ask about each of the services interactively. A bit annoying but should at least temporarily solve the issue

Reply:
Let's specify that the user will be prompted about each service in the message.